src/app/plugins/register.plugin.ts

```diff
@@ -99,24 +99,30 @@ export default class RegisterPlugin extends Plugin {
       }
     });
 
+    const authorData = {
+      name: message.member?.displayName ?? '',
+      iconURL: message.author.avatarURL() ?? '',
+    };
+    const shouldShowAuthor = Constants.ShouldShowAuthorOnRegister;
+
     if (validClasses.length > 0) {
       // List of channel links, one per line
       const validChannels = validClasses
         .map((validClass) => {
           return this.container.classService.findClassByName(validClass);
         })
         .join('\n');
+
+      const embed = new MessageEmbed()
+        .setTitle('Successfully registered')
+        .setDescription(validChannels)
+        .setColor('#a3be8c');
+      if (shouldShowAuthor) {
```

Comment:
```suggestion
      if (Constants.ShouldShowAuthorOnRegister) {
```

Just use the constant here instead of making a redundant variable